stackslib/src/config/mod.rs

```diff
@@ -3420,40 +3538,41 @@ pub struct ConnectionOptionsFile {
     ///
     /// Setting this to `true` disables these restrictions, which can be useful for local testing
     /// environments or fully private network deployments.
-    ///
-    /// Default: `false`.
+    /// ---
+    /// @default: `false`
     pub private_neighbors: Option<bool>,
     /// HTTP auth password to use when communicating with stacks-signer binary.
     ///
     /// This token is used in the `Authorization` header for certain requests.
     /// Primarily, it secures the communication channel between this node and a connected
     /// `stacks-signer` instance.
     ///
-    /// **Requirement:** This field **must** be configured if the node needs to receive block
-    /// proposals from a configured `stacks-signer`event_observer via the `/v3/block_proposal` endpoint. The value
-    /// must match the token configured on the signer.
-    ///
     /// It is also used to authenticate requests to `/v2/blocks?broadcast=1`.
-    ///
-    /// Default: `None` (authentication disabled for relevant endpoints).
+    /// ---
+    /// @default: `None` (authentication disabled for relevant endpoints)
+    /// @notes:
+    ///   - **Requirement:** This field **must** be configured if the node needs to receive
+    ///     block proposals from a configured `stacks-signer` event_observer via the
```

Comment:
```suggestion
    ///     block proposals from a configured `stacks-signer` [`event_observer`] via the
```
for consistency, since we're talking about a specific section of the toml config that is required for this